src/main/java/com/hivemq/codec/decoder/mqtt5/Mqtt5ConnectDecoder.java

```diff
@@ -64,6 +71,7 @@ public Mqtt5ConnectDecoder(
             final @NotNull FullConfigurationService configurationService) {
         super(mqttConnacker, configurationService, clientIds);
         this.hiveMQId = hiveMQId;
+        this.maxMessageExpiryInterval = configurationService.mqttConfiguration().maxMessageExpiryInterval();
```

Comment:
this can be removed

Reply:
thanks, this kept reappearing in EE too when I resolved conflicts; somehow, it got through the patch